iree/compiler/Codegen/LLVMCPU/LLVMCPUTileFuseAndVectorizeLinalgTensorOps.cpp

```diff
@@ -255,6 +254,23 @@ void LLVMCPUTileFuseAndVectorizePass::runOnOperation() {
     });
   }
 
+  // Apply vector unroll
+  {
+    RewritePatternSet vectorUnrollPatterns(context);
+    vector::populateVectorUnrollPatterns(
+        vectorUnrollPatterns, vector::UnrollVectorOptions().setNativeShape(
+                                  config.getNativeVectorSizeVals()));
```

Comment:
I dont know what the native vector size here should be `{16, 16, 16}` seems off. we want `K` dimension to be `16` for vector loads and such. `N` could be 16 for vector loads of `B` matrix. Maybe `{1, 16, 16}` will help with register pressure overall. Fine to do later as well.